app/middleware/renderer.js

```diff
@@ -2,11 +2,11 @@ function results(req, res) {
   res.render('results');
 }
 
-function findHelp(req, res) {
-  res.render('find-help');
+function searchYourGP(req, res) {
```

Comment:
Agree with searchForYourGP, will make that change - I think camelCase all the way is what I'm going for the moment(functionGp)